SubscribeIT/ngDesk
# Changed the subject for daily and monthly signup email

#### Reference to issue (Required)

Closes #354 

#### Checklist (Required)


#### Description (Required)

Changed the subject for daily and monthly signup email

#### Manual Test Preformed (Required)
List the manual tests that were preformed and the steps to preform them  
**There must be multiple test cases provided, each test case should include the exact and detailed steps to replicate the case**
**I have included 3, test cases here but there will be many time when more cases are required**


##### Test Case 1
**Name: POST FEW NEW COMPANY  WITH DIFFERENT SUBDOMAIN**

- go to postman.
- given post call URL:[https://dev1.ngdesk.com/api/ngdesk-company-service-v1/company].
- added proper payload for post call for posting new company
- perform post
- able to post new company without any errors
- able to login in ngdesk with valid credential

##### Test Case 2
**Name:TEST FOR DAILY COMPANY SIGNUP**

- go to company service in eclipse
- go to  package com.ngdesk.company.dao -> CompanySignUpJobservice
- in sendEmail put log for subject
- go to DailyCompanySignupJob in package com.ngdesk.company.jobs;
- give @Scheduled(fixedRate = 6000) and comment cron Schedule
- comment the return of environment check
- able to see logs for proprer subject in console

##### Test Case 3
**Name:TEST FOR MONTHLY COMPANY SIGNUP**

- go to company service in eclipse
- go to  package com.ngdesk.company.dao -> CompanySignUpJobservice
- in sendEmail put log for subject
- go to MonthlyCompanySignupJob in package com.ngdesk.company.jobs;
- give @Scheduled(fixedRate = 6000) and comment cron Schedule
- comment the return of environment check
- able to see logs for proper subject in console

#### List of General Components affected (Required)

**COMPANY SERVICE AND REST**

####Screenshots
![Screenshot from 2022-02-02 11-09-11](https://user-images.githubusercontent.com/89505098/152100174-6514e07e-cee9-49da-a270-b25666808272.png)
